src/media/css/lib/mixins.styl

```diff
@@ -186,8 +186,24 @@ category-icon-position($slug, $scale=1) {
     background-position: ($positions[0] * $cat-icon-size-default * $scale * -1) ($positions[1] * $cat-icon-size-default * $scale * -1);
 }
 
+ltr() {
+    .html-ltr & {
```

Comment:
I was thinking the same thing. We can revisit that later since we've got the mixin. I'm not sure of there's a performance difference or not. I'm assuming not but we can cross that bridge later.